internal/ociregistry/server.go

```diff
@@ -0,0 +1,995 @@
+package ociregistry
```

Comment:
Package comment

And I think we should move this to `internal/pkg/ociregistry` since its not strictly Waypoint-specific (similar to `internal/pkg/epinject`. Now that I type that out loud, maybe we should do `internal/pkg/epinject/ociregistry` since this is really focused on the entrypoint injection? Perhaps.

Reply:
Done!